meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn.service

```diff
@@ -9,4 +9,4 @@ RemainAfterExit=yes
 ExecStart=@BINDIR@/prepare-openvpn
 
 [Install]
-WantedBy=resin.target
```

Comment:
I've stripped the config-json change from this PR as I don't think it's strictly necessary to be in the same PR (we'll see how the tests feel about that though..) but it or a similar form will need to go in for us to transition to dedicated provisioning keys (this PR still requires a user-key to provision with but will then happily switch to a device key)